src/app/pages/send-tokens/components/send-form-inner.tsx

```diff
@@ -93,8 +93,8 @@ export function SendFormInner(props: SendFormInnerProps) {
       <Suspense fallback={<></>}>
         <AmountField error={errors.amount} value={values.amount || 0} />
       </Suspense>
-      <RecipientField error={errors.recipient} value={values.recipient} />
-      {selectedAsset?.hasMemo && <MemoField value={values.memo} error={errors.memo} />}
+      <RecipientField error={errors.recipient} value={values.recipient || ''} />
+      {selectedAsset?.hasMemo && <MemoField value={values.memo || ''} error={errors.memo} />}
```

Comment:
What's the rationale here for making all the form values optional? How did we manage to not have to do this before?

Reply:
If there are different fields in both, it'd be great if we could separate them into two interfaces